react/week3/todo3/src/Components/TodoItems.js

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from "react";
+
+let todos = [
+  {
+    id: 1,
+    description: "Get out of bed",
+    isDone: false,
+  },
+  {
+    id: 2,
+    description: "Brush teeth",
+    isDone: false,
+  },
+  {
+    id: 3,
+    description: "Eat breakfast",
+    isDone: false,
+  },
+  {
+    id: 4,
+    description: "random text",
+    isDone: false,
+  },
+];
```

Comment:
You're supposed to get the array of todos from the api.
Check the outline here:
https://github.com/HackYourFuture-CPH/React/blob/master/week3/homework.md#outline